packages/assets/styles/global.css

```diff
@@ -213,6 +213,22 @@ som fra core/og muligen import fra alla andra komponenter
   border-radius: 6px;
 }
 
+.vis-item.vis-range.avvistPeriode {
+  background-color: rgb(186 58 38 / 50%);
+  border-color: rgb(201 201 201 / 50%);
+  border-radius: 8px;
+}
+
+.vis-item.vis-range.laastPeriode {
+  border-radius: 8px;
+}
+
+.vis-item.vis-range.godkjentPeriode {
+  background-color: rgb(6 137 58 / 50%);
+  border-color: rgb(201 201 201 / 50%);
+  border-radius: 8px;
+}
+
```

Comment:
Et par generelle spørsmål:

Bør vi vurdere å gå vekk fra en global css fil? 
Og burde vi vurdere å bevege oss mot en enkelt type casing av klassenavn (her har vi f.eks. både kebab og snake) på klassenavn?
Og finnes det noen design tokens vi kunne brukt for farger og border-radiuser?